student.rb

```diff
@@ -1,6 +1,9 @@
 require './person'
 require './classroom'
+
 class Student < Person
+  attr_reader :classroom
+
   def initialize(classroom)
```

Comment:
- [ ] also the `def initialize(classroom)` is missing the other parameter like age & name & parent_permission= true

![Capture d’écran 2022-05-27 113649](https://user-images.githubusercontent.com/61562626/170684015-f6132b0d-d339-4b96-b747-392b6acb8e5d.png)

Reply:
Hi @dasileker, i will fix it but it's same to a mentioned above comment